buildSrc/src/main/groovy/bio.terra.catalog.java-common-conventions.gradle

```diff
@@ -41,15 +41,15 @@ repositories {
 
 dependencies {
     compileOnly 'com.github.spotbugs:spotbugs-annotations:4.7.2'
-    implementation 'io.swagger.core.v3:swagger-annotations:2.2.0'
-    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.31'
-
+    implementation 'io.swagger.core.v3:swagger-annotations:2.2.12'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.47'
     implementation 'org.slf4j:slf4j-api'
+    implementation 'ch.qos.logback:logback-classic:1.4.14'
+    implementation 'ch.qos.logback:logback-core:1.4.14'
```

Comment:
It would be nice adding a comment to these pins saying that they can be unpinned following an upgrade to Spring Boot >= 3.2.1 (available tomorrow, hopefully!)